### ISSUE TITLE
F.normalize(self.weight)   --->          F.normalize(self.weight, dim=0)  ?

### ISSUE THREAD

**Issue by xialuxi** [ronghuaiyang/arcface-pytorch#30]

**Comment by ronghuaiyang** [comment]
see #1
